data/json/items/gun/50.json

```diff
@@ -31,8 +39,16 @@
     "copy-from": "rifle_auto",
     "looks_like": "modular_ar15",
```

Comment:
```suggestion
    "looks_like": "m107a1",
```

Reply:
I figuered this way it at least looks more like a 50 cal weapon if there is a .50 cal rifle sprite and if not it still falls back to AR-15